docs/04-operation-guides/security/sec-03-access-kyma.md

```diff
@@ -2,9 +2,9 @@
 title: Access Kyma securely
 ---
 
-As a user, you can access Kyma using the following:
+As a user, you can access Kyma using the following user interfaces:
 
-- Kyma Dashboard, which allows you to view, create, and manage your resources.
+- [Kyma Dashboard](../../../01-overview/main-areas/ui/ui-01-GUI.md), which allows you to view, create, and manage your resources.
```

Comment:
```suggestion
- [Kyma Dashboard](../../01-overview/main-areas/ui/ui-01-GUI.md), which allows you to view, create, and manage your resources.
```

There's a [comment](https://github.com/kyma-project/kyma/pull/12277#issuecomment-933677445) on your PR which still shows one broken [link](https://storage.googleapis.com/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/12277/pre-main-kyma-governance/1445071803855671296/build-log.txt).

Reply:
Thank you for spotting this!